paddle/fluid/framework/details/multi_devices_graph_builder.cc

```diff
@@ -170,6 +170,7 @@ size_t MultiDevSSAGraphBuilder::GetAppropriateDeviceID(
     const std::vector<std::string> &var_names) const {
   int64_t numel_sum = 0;
   for (auto var_name : var_names) {
+    if (all_vars_.find(var_name) == all_vars_.end()) continue;
```

Comment:
`continue` will cause the following `ENFORCE_NOT_NULL` doesn't work, are we sure do that?